NEWS

```diff
@@ -800,8 +800,8 @@ CHANGES WITH 257 in spe:
         Mauri de Souza Meneguzzo, Maximilian Wilhelm, Merlin Jehli,
         Michael Ferrari, Michal Koutný, Michal Sekletár,
         Michele Dionisio, Michiel, Mickaël Salaün, Mike Gilbert,
-        Mike Yuan, MkKvcs, Nick Cao, Nick Rosbrook, Nils K, Nova840,
-        Oğuz Ersen, Pavel Borecki, PavlNekrasov, Peter Hutterer,
+        Mike Yuan, MkKvcs, Nick Cao, Nick Owens, Nick Rosbrook, Nils K,
```

Comment:
Such change will be easily lost when updating the list by the script before the next rc. Please add a short entry before the list of contributors, e.g. `Special thanks` section with short explanation with his name.

Reply:
Ah, I didn't realize this keeps getting regenerated wholesale with every rc.  Will do.